packages/terra-form-select/src/native-select/NativeSelectField.jsx

```diff
@@ -0,0 +1,204 @@
+   */
+  isInvalid: PropTypes.bool,
+  /**
+   * Additional attributes to spread onto the label.
+   */
+  // eslint-disable-next-line react/forbid-prop-types
+  labelAttrs: PropTypes.object,
+  /**
+   * Set the max-width of a field using `length` or `%`.  Best practice recommendation to never exceed
+   * a rendered value of 1020px. _(Note: Providing custom inline styles will take precedence.)_
+   */
+  maxWidth: PropTypes.string,
+  /**
+   * Callback function triggered when the select value changes. function(value)
+   */
+  onChange: PropTypes.func,
+  /**
+   * The array of select options and opt groups.
+   */
+  options: PropTypes.arrayOf(PropTypes.oneOfType([optionPropType, optGroupPropType])),
+  /**
+   * Placeholder data. The presence of this data object is used as the indicator for whether or not a placeholder is used.
+   */
+  placeholder: PropTypes.shape({
+    /**
+     * Whether the placeholder is included as an option to clear selection.
+     */
+    allowClear: PropTypes.bool,
+    /**
+     * Optional override of the default display.
+     */
+    display: PropTypes.string,
+    /**
+     * Optional override of the default value.
+     */
+    value: PropTypes.string,
+  }),
+  /**
+   * Whether the field is required.
+   */
+  required: PropTypes.bool,
+  /**
+   * Additional attributes to spread onto the select.
+   */
+  // eslint-disable-next-line react/forbid-prop-types
+  selectAttrs: PropTypes.object,
+  /**
+   * The Select identifier. Links the htmlFor of the field to the select identifier.
+   */
+  selectId: PropTypes.string.isRequired,
+  /**
+   * Whether to append the 'optional' label to a non-required field label.
+   */
+  showOptional: PropTypes.bool,
+  /**
+   * The value of the select. Can be a string or number.
+   */
+  value: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
+};
+
+const defaultProps = {
+  disabled: false,
+  hideRequired: false,
+  isIncomplete: false,
+  isInline: false,
+  isInvalid: false,
+  isLabelHidden: false,
+  required: false,
+  showOptional: false,
+};
+
+const NativeSelectField = ({
+  defaultValue,
+  disabled,
+  error,
+  help,
+  hideRequired,
+  isIncomplete,
+  isInline,
+  isLabelHidden,
+  isInvalid,
+  label,
+  labelAttrs,
+  maxWidth,
+  onChange,
+  options,
+  placeholder,
+  required,
+  selectAttrs,
+  selectId,
+  showOptional,
+  value,
+  ...customProps
+}) => {
+  const helpText = (
+    <span>
+      {help}
+    </span>
+  );
+
+  let ariaDescriptionIds;
+  if (help && error && isInvalid) {
+    ariaDescriptionIds = `${selectId}-error ${selectId}-help`;
+  } else {
+    if (help) {
+      ariaDescriptionIds = `${selectId}-help`;
+    }
+
+    if (error && isInvalid) {
+      ariaDescriptionIds = `${selectId}-error`;
+    }
+  }
+
+  return (
+    <Field
+      {...customProps}
+      label={label}
+      labelAttrs={labelAttrs}
+      error={error}
+      help={helpText}
```

Comment:
Should this be conditionally added if help text is provided or do we want the empty span there for placeholder space?

Reply:
Updated.